extensions/amp-lightbox-gallery/0.1/service/lightbox-manager-impl.js

```diff
@@ -365,7 +371,19 @@ export class LightboxManager {
   }
 
   /**
-   * The function is not implemented yet. Fake for testing.
+   * Gets the duration of a supported video element
+   * @param {!Element} element
+   * @returns {!Promise}
+   * @private
+   */
+  getVideoTimestamp_(element) {
+    return VIDEO_TAGS[element.tagName] ?
+      element.getImpl().then(videoPlayer => {
+        return videoPlayer.getDuration();
```

Comment:
nit: If you omit the curly braces, the fat arrow will implicitly return the result of the expression.